presto-native-execution/presto_cpp/main/PrestoExchangeSource.h

```diff
@@ -250,6 +250,16 @@ class PrestoExchangeSource : public velox::exec::ExchangeSource {
   const std::string host_;
   const uint16_t port_;
   const folly::SSLContextPtr sslContext_;
+  // If false, we directly build serialized presto page from iobufs allocated by
```

Comment:
Nit : State the behavior if true before the if false.

Reply:
Updated.